### PR TITLE
SC-6391 Bugfix mobile header menu

### PR DESCRIPTION
**Pull request by abralurrahman**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.hpi-schul-cloud.org/browse/SC-6391

## Screenshots of UI changes
Screenshot 2020-08-26 at 12 11 48
Update

UpdateTopVar

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by seulki-lee-cloud** [comment]
could you please share with me fixed version? as screenshot?

**Review by monaweitzenberg — CHANGES_REQUESTED** [review]
image
iPhone5 seems not working, please check

**Comment by ABialoskorski** [comment]
> iPhone5 seems not working, please check

it happens sometimes when You change device in chrome (inspect mode), try some other devices and get back to iphone 5, sometimes it needs few tries to work.

File: static/styles/homepage/homepage.scss [hunk 1]
```diff
@@ -49,3 +49,4 @@ h1.section-title {
 video {
 	border: 1px solid var(--color-white);
 }
+
```
Comment (ABialoskorski) [inline static/styles/homepage/homepage.scss:52]: please delete this change here

**Review by monaweitzenberg — APPROVED** [review]
test again in firefox. went well. good job!!! thx :)

**Review by ABialoskorski — APPROVED** [review]

**Review by mwolowyk-cap — APPROVED** [review]

**Pull request merged**